src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestStream.cs

```diff
@@ -110,7 +110,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
 
             if (count > buffer.Length - offset)
             {
-                throw new ArgumentException(nameof(buffer));
+                throw new ArgumentException(SR.Argument_BufferNotLongEnough, nameof(buffer));
```

Comment:
Should this have been `ArgumentOutOfRangeException`?  I think there was a resource string mentioning count/offset, but the exception tagged the buffer...